src/Campaigns/Actions/LoadCampaignDetailsAssets.php

```diff
@@ -33,14 +34,17 @@ public function __invoke()
             true
         );
 
-        $defaultForm = $campaign->defaultForm();
-        $defaultFormTitle = Utils::isV3Form($defaultForm->id) ? $defaultForm->settings->formTitle : $defaultForm->title;
         wp_localize_script($handleName, 'GiveCampaignDetails',
             [
                 'adminUrl' => admin_url(),
                 'currency' => give_get_currency(),
                 'isRecurringEnabled' => defined('GIVE_RECURRING_VERSION') ? GIVE_RECURRING_VERSION : null,
-                'defaultForm' => $defaultFormTitle,
+                'donationForms' => array_map(function(DonationForm $form) {
+                    return [
+                        'id' => $form->id,
+                        'title' => $form->hasProperty('settings') ? $form->settings->formTitle : $form->title,
+                    ];
+                }, $campaign->forms()->getAll()),
```

Comment:
We need a way to load this form list beyond the first page load because it can get inaccurate easily depending of how user add new forms to the campaign - check my attached video for more details.

Reply:
Is a list of forms available for re-use from the Campaign Forms List Table?

Reply:
@kjohnson  I don't know if our List Table component provides some way to access the list of available items through a JS object or something like that, but I think not, even because the list table can have pagination wich makes me think again that this field to update the default form trough the settings tab is not a good idea because in situations where a campaign has a massive amount of forms in a single campaign, populate this dropdown can become a problem.